addons/account_edi_proxy_client/demo/demo.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" ?>
+<odoo>
+
```

Comment:
@bfr-o Would you check for the name of this file?  I think we also put demo data more in data map itself and maybe call it config_demo.xml or something (maybe check guidelines)